internal/core/application/operator_service.go

```diff
@@ -1843,6 +1844,15 @@ func (o *operatorService) splitFeeFragmenterFunds(
 		}
 		return
 	}
+	if int(amountPerAsset[lbtc]) < MinFeeFragmenterAmount {
```

Comment:
In light of #517 we must maybe check the fee account. Depositing low amounts of bitcoins, does not matter anymore (ie. should always work).

The only argument is in favor of not being economically meaningful, but this should be either a warning or a convenience alert in the UI